project/src/main/webapp/event-details.js

```diff
@@ -97,22 +100,24 @@ async function showVolunteeringOpportunities(
  * @return {string}
  */
 function getListItemForOpportunity(
-    opportunityId, name, numSpotsLeft, requiredSkills,
-    volunteers, eventHost, isHost) {
-  requiredSkillsText =
-      requiredSkills.length ? requiredSkills.toString() : 'None';
-  volunteersText =
-      volunteers.length ? volunteers.toString() : 'None';
+    opportunityId, name, numSpotsLeft, requiredSkills, volunteers, eventHost,
+    isHost) {
+  const volunteersText = volunteers.length ?
+      volunteers.toString().replace(/,/g, ', ') :
+      'None';  // remove
```

Comment:
Do you mean "remove" is a TODO or what this code is doing right now?

Reply:
done remove comment.